libc/newhdrgen/yaml_to_classes.py

```diff
@@ -143,6 +196,19 @@ def main(yaml_file, h_def_file, output_dir):
         default=".",
         help="Directory to output the generated header file",
     )
+    parser.add_argument(
+        "--add_function",
+        nargs=6,
+        metavar=(
+            "NAME",
```

Comment:
this should probably be in the same order that you'd find them in a function definition (i.e. `RETURN_TYPE NAME (ARGUMENTS)...`)

Reply:
fixed!

Reply:
this seems like the same order as before, maybe the formatter reordered them?